config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "process.env.DB_USER",
+    "password": "process.env.DB_PASSWORD",
```

Comment:
The value for `password` should be accessed using `process.env.DB_PASSWORD` without quotes to correctly retrieve the environment variable.